gha.env

```
@@ -0,0 +1,7 @@
+DOCKERHUB_ORG_NAME=dockerdevrel
+IMAGE_NAME_SUFFIX=catalog-service-node-gha
```

There are two separate image names being used here, one of which is using the `-gha` suffix and the other omitting that.

The long term idea is to, once we have further insights on repo storage sizes, be able to show not only how DBC makes image builds faster, but also dramatically reduces the size of the repos by reusing layers.

---

Can we do this with tags? If this is the same application, it should probably be the same image?

---

```suggestion
IMAGE_NAME_SUFFIX=catalog-service-node
```

Removing the GHA tag here and then putting it back on the GHA-based workflow. It might be worth considering removing that workflow, but will save that conversation for a separate PR.